tests/conftest.py

```diff
@@ -1,17 +1,60 @@
+import base64
 import os
 
 import pytest
+from six.moves.urllib.parse import parse_qs, urlparse
 
-from wptserve import server
+from wptserve import (
+    handlers,
+    request,
+    routes as default_routes,
+    server,
+)
+
+class Authentication(request.Authentication):
+    """Override the faulty decode_basic method in request.Authentication
+       The original method uses base64.decodestring which gives an error if a str
+       is passed instead of a bytestring.
+       base64.b64decode works on both str and bytestring"""
+    def decode_basic(self, data):
+        decoded_data = base64.b64decode(data).decode()
+        return decoded_data.split(":", 1)
+
+@handlers.handler
+def http_auth_handler(req, response):
```

Comment:
Please name it `basic_auth_handler` given that there are other authentication methods we would like to support in the future.